cognite/client/_api/simulators/routines.py

```diff
@@ -226,22 +226,58 @@ def list(
             filter=routines_filter.dump(),
         )
 
+    @overload
     def run(
         self,
+        *,
         routine_external_id: str,
         inputs: Sequence[SimulationInputOverride] | None = None,
         run_time: int | None = None,
         queue: bool | None = None,
         log_severity: Literal["Debug", "Information", "Warning", "Error"] | None = None,
         wait: bool = True,
         timeout: float = 60,
+    ) -> SimulationRun: ...
+
+    @overload
+    def run(
+        self,
+        *,
+        routine_revision_external_id: str,
+        model_revision_external_id: str,
+        inputs: Sequence[SimulationInputOverride] | None = None,
+        run_time: int | None = None,
+        queue: bool | None = None,
+        log_severity: Literal["Debug", "Information", "Warning", "Error"] | None = None,
+        wait: bool = True,
+        timeout: float = 60,
+    ) -> SimulationRun: ...
+
+    def run(
+        self,
+        routine_external_id: str | None = None,
```

Comment:
I think this should also allow you to take in a full run object.
```suggestion
        routine_external_id: str | SimulationRunWrite | None = None,
```

This will allow you to
```python
client.simulators.run(my_run_obj). 
```

Reply:
we wanted to keep this one simple, also, can be added later, anyway?